src/xo-app/home/index.js

```diff
@@ -463,6 +558,11 @@ export default class Home extends Component {
     return customFilters[this._getType()]
   }
 
+  _getNoResourceSets = createSelector(
+    () => this.props.resourceSets,
+    isEmpty
+  )
```

Comment:
No need to use a memoized selector for `isEmpty`: this function is fast (`O(1)`) and do not create new values.